src/UI/Implementation/Component/Input/Field/SwitchableGroup.php

```diff
@@ -87,6 +86,9 @@ public function withValue($value) {
 	 */
 	public function getValue() {
 		$key = Input::getValue();
+		if(is_null($key)){
+			return $key;
```

Comment:
```suggestion
			return null;
```
The value is supposed to be `null` here, `$key` just acidentially is `null` here as well.